include/uv-win.h

```diff
@@ -78,6 +78,7 @@ typedef intptr_t ssize_t;
 # define SIGABRT_COMPAT       6
 #endif
 
+UV_EXTERN int uv_translate_win32_error(int sys_errno);
```

Comment:
I remember that, but we don't have a mapping for Windwos posix style errnos, so I think i'd go with `uv_translate_sys_error` at the moment.